.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This is a cool vscode feature that allows you to add some custom debugging tools. This adds a new debugger starting point named "Run show-customers command" - you can access this via the "Run and Debug" tab. I'll show you how to use it on Monday. 

<img width="918" alt="image" src="https://github.com/vivek-shah-13/store/assets/2894436/4a106278-11e8-4cb1-b816-64ff353120d8">